packages/bvaughn-architecture-demo/components/sources/PointPanel.tsx

```diff
@@ -53,45 +53,54 @@ export default function PointPanel({ className, point }: { className: string; po
           placeholder="Content..."
           value={editableContent}
         />
-        <label className={styles.Label}>
-          <input
-            checked={point.shouldBreak}
-            onChange={event => editPoint(point.id, { shouldBreak: event.currentTarget.checked })}
-            type="checkbox"
-          />
-          <Icon className={styles.LabelIcon} type="pause" />
-        </label>
-        <label className={styles.Label}>
-          <input
-            checked={point.shouldLog}
-            onChange={event => editPoint(point.id, { shouldLog: event.currentTarget.checked })}
-            type="checkbox"
-          />
-          <Icon className={styles.LabelIcon} type="print" />
-        </label>
-        <small>
-          ({point.location.line}:{point.location.column})
-        </small>
+        {false && (
+          <>
+            {" "}
+            <label className={styles.Label}>
+              <input
+                checked={point.shouldBreak}
+                onChange={event =>
+                  editPoint(point.id, { shouldBreak: event.currentTarget.checked })
+                }
+                type="checkbox"
+              />
+              <Icon className={styles.LabelIcon} type="pause" />
+            </label>
+            <label className={styles.Label}>
+              <input
+                checked={point.shouldLog}
+                onChange={event => editPoint(point.id, { shouldLog: event.currentTarget.checked })}
+                type="checkbox"
+              />
+              <Icon className={styles.LabelIcon} type="print" />
+            </label>
+            <small>
+              ({point.location.line}:{point.location.column})
+            </small>
+          </>
+        )}
         <button
           className={styles.SaveButton}
           disabled={!isConditionValid || !isContentValid || !hasChanged}
           onClick={() =>
             editPoint(point.id, { condition: editableCondition, content: editableContent })
           }
         >
-          <Icon className={styles.SaveButtonIcon} type="save" />
+          <Icon className={styles.SaveButtonIcon} type="comment" />
         </button>
       </div>
-      <div className={styles.Row}>
-        <input
-          className={styles.Input}
-          data-test-id={`PointPanelInput-${point.location.line}-condition`}
-          onChange={event => setEditableCondition(event.currentTarget.value || null)}
-          onKeyDown={onKeyDown}
-          placeholder="Condition..."
-          value={editableCondition || ""}
-        />
-      </div>
+      {false && (
```

Comment:
This breaks editing the conditional, which is an important part of the test harness' functionality